app/views/api/applications/new.html.slim

```diff
@@ -0,0 +1,5 @@
+h2 New Application
+
+div id="new-application-form" data=(new_application_form_metadata(provider: @provider, service: @service))
```

Comment:
Why don't you just use  `current_account` instead of `@provider`
No need of defining `@provider` in the controller

It is defined as a helper in https://github.com/3scale/porta/blob/06215fac9733963435a266c60ead0bde51b3dff8/app/lib/authenticated_system.rb#L12